Backpack/Alert/Classes/BPKAlertConfiguration.m

```diff
@@ -0,0 +1,56 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "BPKAlertConfiguration.h"
+
+@implementation BPKAlertConfiguration
+
+- (instancetype)init
+{
+    self = [super init];
+    if (self) {
+        NSAssert(NO, @"Please call the custom init or the static methods");
+    }
+    return self;
+}
+
+- (instancetype _Nonnull)initWithCircleColor:(UIColor *)circleColor
+                                        icon:(UIImage *)iconImage
+                                   titleText:(NSString *)titleText
+                             descriptionText:(NSString *)descriptionText
+                        buttonConfigurations:(NSArray<BPKAlertButtonConfiguration *> *)buttonConfigurations
+                                   hasShadow:(BOOL)hasShadow
+                     doneButtonConfiguration:(BPKAlertDoneButtonConfiguration *)doneButtonConfiguration
+                          faderConfiguration:(BPKAlertFaderConfiguration *)faderConfiguration
+                                isFullScreen:(BOOL)isFullScreen {
+    self = [super init];
+    if (self) {
+        _circleColor = circleColor;
```

Comment:
Can we please have private  `readwrite` properties for these?

Reply:
you would like these properties to be mutable and set from the outside?